python/django_bolt/auth/user_loader.py

```
@@ -49,17 +50,24 @@
 # or None when the backend has no user resolution. Built at registration time.
 _resolved_loader_registry: dict[str, Callable[[str, dict | None, bool], Any] | None] = {}
 
-# Global shared thread pool for user loading - avoids expensive pool creation per request
-# Using max_workers=4 to limit resource usage while allowing concurrent user loads
-_user_loader_executor: concurrent.futures.ThreadPoolExecutor | None = None
 
+def _run_orm_blocking(fn: Callable[..., Any], *args: Any) -> Any:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)